src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +50,21 @@
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get the historical price data for the passed in symbol over the passed in range.
+   *
+   * @param symbol stock symbols to get historical data for.
+   * @param range selection of time for which to get historical data.
+   * @return a list of historical price objects for the Symbol that is passed in covering the date
+   *     range.
+   */
+  @GetMapping(value = "${mvc.iex.getHistoricalPricesPath}", produces = {
+      MediaType.APPLICATION_JSON_VALUE})
+  public List<IexHistoricalPrice> getHistoricalPrices(
+      @RequestParam(value = "symbol") final String symbol,
+      @RequestParam(value = "range") final String range,
+      @RequestParam(value = "token") final String token) {
+    return iexService.getHistoricalPricesForSymbol(symbol, range, token);
```

Comment:
I don't think we should require the user of our application to provide the token for IEX